kazoo/tests/conftest.py

```diff
@@ -4,7 +4,10 @@
 
 
 def pytest_exception_interact(node, call, report):
-    cluster = node._testcase.cluster
-    log.error("Zookeeper cluster logs:")
-    for logs in cluster.get_logs():
-        log.error(logs)
+    if hasattr(node._testcase, "cluster"):
+        cluster = node._testcase.cluster
+        log.error("Zookeeper cluster logs:")
+        for logs in cluster.get_logs():
+            log.error(logs)
+    else:
+        log.error("Cluster was not up!")
```

Comment:
Some tests does not make use of the cluster, I think the `else` case can just be omitted here. What do you think?

Reply:
I think you're right. The lack of a cluster attribute is a marker that the test was not meant to have a cluster.

I'll remove the else.